src/api/transaction.rs

```diff
@@ -265,7 +265,7 @@ pub fn contract_address_string(contract_addr: &ContractAddress) -> String {
     format!("{}{}_{}", ACCOUNT_CONTRACT_PREFIX, contract_addr.index, contract_addr.subindex)
 }
 
-pub fn map_transaction(info: &BlockItemSummary) -> Transaction {
+pub fn map_transaction(info: BlockItemSummary) -> Transaction {
```

Comment:
Doesn't this lead to an unnecessary copy?

Reply:
If the input is never cloned then it does not matter.

It seems there is an unnecerssary clone in block_transactions where the code should be
```
                let tx = finalized_status
                    .into_iter()
                    .next()
                    .ok_or_else(|| {
                        ApiError::InternalServerError(anyhow::anyhow!(
                            "Claimed finalized block is empty."
                        ))
                    })?
                    .1;
```
instead of what it is now.

Reply:
Brilliant, thanks